pom.xml

```diff
@@ -114,6 +114,19 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      
+      
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpcore</artifactId>
+        <version>4.4.14</version>
```

Comment:
The version numbers should probably be made variables like the other dependencies.

Reply:
Done